tests/integrate/Autotest.sh

```diff
@@ -100,16 +104,30 @@ check_out(){
 		# deviation should be positively defined
 		#--------------------------------------------------
 		if [ ! -n "$deviation" ]; then
-            echo -e "\e[0;31m[ERROR     ] Fatal Error: key $key not found in output.\e[0m"
+			echo -e "\e[0;31m[ERROR     ] Fatal Error: key $key not found in output.\e[0m"
 			let fatal++
 			fatal_case_list+=$dir'\n'
 			break
-        else
+		else
 			if [ $(echo "sqrt($deviation*$deviation) < $threshold"|bc) = 0 ]; then
+				if [ $key == "totalforceref" -o $key == "totalstressref"]; then
```

Comment:
```suggestion
				if [ $key == "totalforceref" -o $key == "totalstressref" ]; then
```

@Zhuxuegang2022 Actually this if statement is not necessary by "if... elif ... else".